server/src/main/java/org/cloudfoundry/identity/uaa/util/AlphanumericRandomValueStringGenerator.java

```diff
@@ -0,0 +1,44 @@
+package org.cloudfoundry.identity.uaa.util;
+
+import java.security.SecureRandom;
+import java.util.Random;
+
+public class AlphanumericRandomValueStringGenerator {
```

Comment:
In 74.5.x, `AlphanumericRandomValueStringGenerator` extends `org.springframework.security.oauth2.common.util.RandomValueStringGenerator`.  Any specific reason not doing the same?

Reply:
@hsinn0 Sonar gave warnings to not extend a class that is Deprecated and Sonar checks failed.

Reply:
I see.  In fact, we know that the whole `spring-security-oauth2` is deprecated and have a story to replace with different one.  Because the whole UAA project has dependency to the EOL-ed `spring-security-oauth2` package, it practically does not really matter for this one class to depend on a deprecated class.
Anyhow, this change will do the job as it is, so I am fine with it then.

Reply:
Yes agree. Since we modified this file, sonar was scanning and complained about this specific class usage.